spec/javascripts/angular/services/url_generator_spec.js

```diff
@@ -17,7 +17,7 @@ describe('URLGenerator', function() {
   it('allows urls with custom paths, with trailing slash', function() {
     ['http://promdash.server.com/prometheus/', 'http://promdash.com/'].forEach(function(s) {
       ['/api/query_range', '/api/query', '/api/metrics', '/arbitrary/endpoint'].forEach(function(ep) {
-        var url = urlGenerator(s, ep);
+        var url = urlGenerator(s, ep, {});
         expect(url).toEqual(s.substring(0, s.length - 1) + ep);
       });
     });
```

Comment:
could you add a test checking that the interpolation is going into the url correctly?

```javascript
  it('interpolates into urls', function() {
    var s = 'http://promdash.{{zone}}.com/prometheus';
    ['aa', 'bb', 'cc', 'dd'].forEach(function(zone) {
      var url = urlGenerator(s, '', {zone: zone});
      expect(url).toEqual('http://promdash.' + zone + '.com/prometheus');
    });
  });
```